Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "fobnail"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[target.'cfg(target_os = "none")'.dependencies]
+pal_nrf = { path = "pal/pal_nrf" }
```

Comment:
@arturkow2000 Are you sure that the nrf dependencies are bot pulled in for the PC build? I think I tried to build without nrf-hal repo and it failed (even when I build for PC target).

Reply:
@macpijan Seems like Cargo always parses metadata for all crates, even those not included by current target, probably nothing we can do here.

Build log reveals that `nrf-hal` is not built when building for PC.
```
   Compiling memchr v2.4.1
   Compiling libc v0.2.108
   Compiling log v0.4.14
   Compiling cfg-if v1.0.0
   Compiling quick-error v1.2.3
   Compiling regex-syntax v0.6.25
   Compiling termcolor v1.1.2
   Compiling managed v0.7.2
   Compiling byteorder v1.4.3
   Compiling bitflags v1.3.2
   Compiling humantime v1.3.0
   Compiling aho-corasick v0.7.18
   Compiling atty v0.2.14
   Compiling smoltcp v0.7.5
   Compiling regex v1.5.4
   Compiling env_logger v0.7.1
   Compiling pretty_env_logger v0.4.0
   Compiling pal_pc v0.1.0 (/hdd/projekty/fobnail/fobnail/pal/pal_pc)
   Compiling fobnail v0.1.0 (/hdd/projekty/fobnail/fobnail)
    Finished dev [unoptimized + debuginfo] target(s) in 9.85s
```